src/Module/NetteDIModule.php

```diff
@@ -129,11 +132,13 @@ private function createContainer(): void
 			$configurator->enableDebugger($logDir);
 		}
 
-		$configurator->addParameters([
+		$parameters = array_merge($this->parameters, [
 			'appDir' => $this->path . ($this->config['appDir'] !== null ? '/' . $this->config['appDir'] : ''),
 			'wwwDir' => $this->path . ($this->config['wwwDir'] !== null ? '/' . $this->config['wwwDir'] : ''),
 		]);
 
+		$configurator->addParameters($parameters);
```

Comment:
This is still very limiting. Please split createContainer into createConfigurator and createContainer. Should work like that: create and configure configurator, run onCreateConfigurator events, create container, run onCreateContainer events